docs/store-operations/pricing/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,93 @@
+# Price Lists - International Enhancements for Multi-Storefront
+
+<Callout type="info">
+  Tax-inclusive and exclusive price lists are in beta. Features may change based on user feedback or internal testing.
+</Callout>
+
+When you sell internationally, you may need your storefront to display prices that either include or exclude tax based on regional expectations. You may need some storefront channels to display prices that include tax, while others display prices that exclude tax.
+
+To localize your storefronts, you can do the following:
+- Specify whether a price list has prices that include or exclude tax when entered.
```

Comment:
Add prices (of your desired price entered with tax) setting to the price list in your desired currency

Reply:
added 👍